wordcount.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
```

Comment:
nice job overall, but you didn't take all types of whitespace into consideration (like tabs, newlines), so the count was off a bit!